docs/configuration/configuration.md

```diff
@@ -451,6 +451,26 @@ metric_relabel_configs:
 # native histogram. If this is exceeded, the entire scrape will be treated as
 # failed. 0 means no limit.
 [ native_histogram_bucket_limit: <int> | default = 0 ]
+
+# Lower limit for the growth factor of one bucket to the next in each native histogram. 
+# The resolution of a histogram with a lower growth factor will be reduced until it is within the limit.
+# To set an upper limit for the schema (equivalent to "scale" in OTel's exponential histograms), use the following factor limits:
+# | growth factor | resulting schema AKA scale |
+# | ---   | --- |
+# | 65536 | -4  |
+# | 256   | -3  |
+# | 16    | -2  |
+# | 4     | -1  |
+# | 2     |  0  |
+# | 1.4   |  1  |
+# | 1.1   |  2  |
+# | 1.09  |  3  |
+# | 1.04  |  4  |
+# | 1.02  |  5  |
+# | 1.01  |  6  |
+# | 1.005 |  7  |
+# | 1.002 |  8  |
```

Comment:
Let's add a line here about the default, like: "0 results in the smallest supported factor (which is currently ~1.0027 or schema 8, but might change in the future)."